### PR TITLE
Fixed learn-more link

### LINKED ISSUE #3153: Learn more link on PI card is incorrect
### Dev Home version

_No response_

### Windows build number

_No response_

### Other software

_No response_

### Steps to reproduce the bug

Click Learn more on the card in Utilities after enabling Project Ironsides in experimental feature settings

### Expected result

This should take you to Dev Home documentation at a minimum, ideally to Project Ironsides docs once they're published
This link should also use a go.microsoft.com link, not an aka.ms link

### Actual result

This goes to microsoft.com using an aka.ms link

### Included System Information

_No response_

### Included Extensions Information

_No response_

### PR DESCRIPTION
**Pull request by andreww-msft**
## Summary of the pull request
We have a fwlink created for the learn-more page, but that page doesn't exist yet. So this fix sets the link to the Feature Exploration page for Project Ironsides in GitHub.  A separate issue will be opened to track updating the link again later, when the new documentation page is available. New issue: https://github.com/microsoft/devhome/issues/3190

Also renamed Seperator.xaml/cs to Separator.xaml/cs, and fixed the alignment of the Pinned toggleswitch in the AddTools control.

## References and relevant issues
- [ ] Closes #3153

### CODE REVIEW

**Review by zadesai — APPROVED** [review]
:shipit:

File: tools/Utilities/src/ViewModels/UtilitiesMainPageViewModel.cs [hunk 1]
```diff
@@ -54,7 +54,7 @@ public UtilitiesMainPageViewModel(IExperimentationService experimentationService
             {
                 Title = stringResource.GetLocalized("ProjectIronsidesTitle"),
                 Description = stringResource.GetLocalized("ProjectIronsidesDesc"),
-                NavigateUri = "https://aka.ms/projectironsides",
+                NavigateUri = "https://github.com/microsoft/devhome/issues/2796",
```
Comment (timkur) [inline tools/Utilities/src/ViewModels/UtilitiesMainPageViewModel.cs:57]: Did I remember Kayla saying we were supposed to use go.microsoft.com links?

Comment (andreww-msft) [reply tools/Utilities/src/ViewModels/UtilitiesMainPageViewModel.cs:57]: Yes, and the fwlink I created is indeed a go.microsoft.com link - but right now it links to a page that doesn't exist yet. This PR points to the same URL as in Project Ironsides' About page. I opened a new issue for a second update to use the fwlink when that page is available.

Comment (zadesai) [reply tools/Utilities/src/ViewModels/UtilitiesMainPageViewModel.cs:57]: How about we update aka.ms/projectironsides to point to https://github.com/microsoft/devhome/issues/2796. And when we have the new page, we can just update the aka.ms/projectironsides link?

Comment (andreww-msft) [reply tools/Utilities/src/ViewModels/UtilitiesMainPageViewModel.cs]: Oh that's much more sensible, thanks! Pushed a new change to use the fwlink. I'll use the other issue to track fixing the fwlink itself when we get the page.

**Review by timkur — APPROVED** [review]
:shipit:

**Pull request merged**